test/clj_kondo/analysis_test.clj

```diff
@@ -577,12 +577,12 @@
        var-definitions)
       (assert-submaps
        '[{:name defmulti}
-         {}
```

Comment:
There used to be a weird invisible character on the line between the defmulti and the defmethod. That character got analyzed as a var-usage, which is what this test line was about. I removed that character, but let me know if you want me to restore it.